po/de.po

```diff
@@ -917,7 +910,7 @@ msgstr "GnuCash Version %s"
 
 #: docs.phtml:60
 msgid "Current <em>stable</em> user documentation"
-msgstr "Letzte veröffentlichte Dokumentation für Benutzer"
+msgstr "Letzte <em>stabile</em> Dokumentation für Benutzer"
```

Comment:
Auch hier war die Absicht, das in der anderen Richtung zu berichtigen, weil {unstable, stable, old stable} nicht so toll ist.